mlflow/metrics/genai/model_utils.py

```diff
@@ -116,6 +115,44 @@ def _call_openai_api(openai_uri, payload):
     except Exception as e:
         raise MlflowException(f"Error response from OpenAI:\n {e}")
 
+    return _convert_from_openai_response(resp)
+
+
+def _call_gateway_api(gateway_uri, payload):
+    from mlflow.gateway import query
+
+    return query(gateway_uri, payload)
+
+
+def _call_deployments_api(deployment_uri, payload):
+    from mlflow.deployments import get_deploy_client, get_deployments_target
+
+    target = get_deployments_target()
+    client = get_deploy_client(target)
+
+    payload = _convert_to_openai_request(payload)
+    resp = client.predict(endpoint=deployment_uri, inputs=payload)
+    return _convert_from_openai_response(resp)
+
+
+def _convert_to_openai_request(payload):
+    payload = {{"candidate_count": "n"}.get(k, k): v for k, v in payload.items()}
+    payload["temperature"] = 2 * payload["temperature"]
+    payload["messages"] = [
+        {
+            "role": "system",
+            "content": """
+         You must return the following fields in your response in two lines, one below the other. Do not add additional new lines.
+score: Your numerical score for the model
+justification: Your reasoning for the score
+         """,
```

Comment:
Should this be part of the v1 prompt somehow and hardcoded in utils? Also do we need this since we have already update the v1 prompt?